helm/botkube/Chart.yaml

```
@@ -1,8 +1,8 @@
 apiVersion: v1
 name: botkube
 home: https://www.botkube.io
-version: v0.9.1
-appVersion: v0.9.1
+version: v0.9.2
+appVersion: v0.9.2
```

The chart version will be updated by release scripts when we publish the release
```suggestion
version: v0.9.1
appVersion: v0.9.1
```